index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microvese</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="body">
+    <!-- <h1>Hello Wordl! we are software developers!</h1> -->
+    <header id="main">
+        <h1 id="title">Dr... Ben And Dr. Alick</h1>
+        <p>The man who saved a billion lives</p>
+        <h2>hello from </h2>
+        <div id="img-div">
+            <img src="img1.jpg" id="image" alt="" srcset="">
+            <p id="img-caption">Dr. Ben And Dr. Alick, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger. </p>
+        </div>
+        
+    </header>
+    
+    <article id="tribute-info">
+        <h3>Here's a time line of Dr. Ben And Dr. Alick, life:</h3>
+        <ul>
+            <li>
+                <p> <strong>1914</strong> - Born in Cresco, Iowa ...modified</p>
```

Comment:
- [ ] Please Avoid the space  between two tags: <p> and <strong> so as to make your code more professional